web/docs/guides/integrations/directus.mdx

```diff
@@ -10,25 +10,25 @@ In this guide, we will demonstrate how to create a new Supabase project, install
 
 ![Supabase App](/img/guides/integrations/directus/supabase-20220608A.webp)
 
-Supabase is an open-source Firebase alternative that provides a PostgreSQL database, storage, authentication, and a dynamic REST API based on your schema. While it is possible to self-host Supabase on your own infrastructure, this article will focus on Supabase Cloud’s Free tier, which is the fastest and easiest way to get started.
+[Supabase](​​https://supabase.com/) is an open-source Firebase alternative that provides a PostgreSQL database, storage, authentication, and a dynamic REST API based on your schema. While it is possible to self-host Supabase on your own infrastructure, this article will focus on Supabase Cloud’s Free tier, which is the fastest and easiest way to get started.
 
 ![Directus App](/img/guides/integrations/directus/directus-20220608A.webp)
 
-Directus is an open-source data platform that layers on top of any SQL database, providing a powerful suite of tools. The Directus Engine provides dynamic REST and GraphQL APIs based on your schema, hooks and automation, authentication and access control, and file transformations. Directus Studio enables engineers and non-technical users alike to browse, manage, and visualize database content through a no-code app.
+[Directus](https://directus.io/) is an open-source data platform that layers on top of any SQL database, providing a powerful suite of tools. The Directus Engine provides dynamic REST and GraphQL APIs based on your schema, hooks and automation, authentication and access control, and file transformations. Directus Studio enables engineers and non-technical users alike to browse, manage, and visualize database content through a no-code app.
 
 ---
 
-Supabase is a suite of open-source tools making Postgres databases, file storage, authentication, and edge functions more accessible to developers of all skill levels. Directus is also developer tooling and additionally provides a Data Studio that is safe and intuitive enough for anyone, including non-technical users, to use. This is the crucial bit that gives the two platforms such a strong “network effect.”
+Supabase is strictly developer tooling, making it great for engineers, but its administration app can be puzzling for non-technical business users. Directus is also developer tooling, but its Data Studio is safe and intuitive enough for anyone to use. This is the crucial bit that gives the two platforms such a strong “network effect.”
```

Comment:
>but its administration app can be puzzling

Since these are our official docs, I don't know if we want to be "putting ourselves down" like this @erondpowell 😂 

Happy for you to make a case for Directus, but probably not at the expense of our brand

Reply:
Heya! Yeah that does read a little funky. These edits were signed off by both teams, so I just copied it over into this PR without really doing another pass, sorry 'bout that! 🙈 Updating now.